src/cascadia/TerminalSettingsEditor/KeyBindingsPage.xaml

```diff
@@ -0,0 +1,17 @@
+﻿<Page
+    x:Class="TerminalSettings.KeyBindingsPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:TerminalSettings"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Grid>
+
+        <TextBox IsEnabled="False" Name=""/>
+        <Button x:Name="asdf" Click="">Click me</Button>

```

Comment:
hope you didn't add asdf to the spellchecker for this

Reply:
![image](https://user-images.githubusercontent.com/57155886/93120566-6b01a600-f678-11ea-8aca-ec704fa0b9a4.png)
funny enough, it already exists in `expect.txt` 😆